src/poprox_storage/repositories/newsletters.py

```diff
@@ -315,6 +315,9 @@ def extract_and_flatten(newsletters: list[Newsletter], include_treatment: bool =
             record["profile_id"] = str(newsletter.account_id)
             record["newsletter_id"] = str(newsletter.newsletter_id)
             record["article_id"] = str(impression.article.article_id)
+            record["article_preview_image_id"] = (
+                str(impression.article.preview_image_id) if impression.article.preview_image_id else None
```

Comment:
```suggestion
                str(impression.preview_image_id) if impression.preview_image_id else ""
```
The rest of the code around this seems to use `""` as null instead of a true `None` -- I vaguely remember @karlhigley running into issues with literal `None` in reviews.

Suggested change also accounts for a change we made recently (adding a `preview_image_id` field to `impression`)

Reply:
The types in each column need to be consistent in order to successfully write Parquet files 👍🏻

Reply:
@kluver 
I’ve updated the code based on the feedback. As of now, I haven’t identified any remaining issues.